### PR TITLE
[DOCS] Refresh transform screenshots with histograms

### PR DESCRIPTION
**Pull request by lcawl**
Related to https://github.com/elastic/kibana/issues/69235

This PR refreshes the screenshots in the transform tutorial (https://www.elastic.co/guide/en/elasticsearch/reference/master/ecommerce-transforms.html) and transform overview (https://www.elastic.co/guide/en/elasticsearch/reference/master/transform-overview.html) to reflect the addition of data grid histograms. 

## Preview

https://elasticsearch_59264.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/ecommerce-transforms.html

### CODE REVIEW

File: docs/reference/transform/ecommerce-tutorial.asciidoc [hunk 1]
```diff
@@ -42,7 +43,7 @@ Go to *Management* > *Elasticsearch* > *Transforms* in {kib} and use the
 wizard to create a {transform}:
 
 [role="screenshot"]
-image::images/ecommerce-pivot1.jpg["Creating a simple {transform} in {kib}"]
+image::images/ecommerce-pivot1.png["Creating a simple {transform} in {kib}"]
```
Comment (peteharverson) [inline docs/reference/transform/ecommerce-tutorial.asciidoc:46]: I would retake this screenshot to use a selection of columns which show off the histogram charts better - specifically try and include some numeric fields. For example:

image

This selection also includes the numeric `taxless_total_price` and `total_quantity` which you use in the example transform. The distribution chart of the country_iso_codes also looks nice!

**Comment by lcawl** [comment]
@elasticmachine run elasticsearch-ci/docs

File: docs/reference/transform/ecommerce-tutorial.asciidoc (hunk 1 above)
Comment (lcawl) [reply docs/reference/transform/ecommerce-tutorial.asciidoc:46]: Thanks @peteharverson I've updated the screenshots with the columns you suggested front and centre

**Review by peteharverson — APPROVED** [review]
New screenshot LGTM!

File: docs/reference/transform/ecommerce-tutorial.asciidoc [hunk 2]
```diff
@@ -1,4 +1,4 @@
-[role="xpack"]
+png[role="xpack"]
```
Comment (lcawl) [inline docs/reference/transform/ecommerce-tutorial.asciidoc:1]:
```suggestion
[role="xpack"]
```

**Pull request merged**